docs/api/draggable.md

```diff
@@ -453,6 +453,7 @@ It is possible for your `<Draggable />` to contain interactive elements. By defa
 - `video`
 - `audio`
 - [`contenteditable`](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/contenteditable) (any elements that are `contenteditable` or are within a `contenteditable` container)
+- Any elements that have a `data-interactive-element` property set to true, or any element that is contained in such element.
```

Comment:
can you add one to `data-attributes.js`

Something like:

`${prefix}-is-interactive-element`

Lets have more of a think about the name

Otherwise this looks good!

Reply:
Hi Alex, thanks for the feedback!

Could you check if the changes I made are the ones you had in mind?

Reply:
I'll try to take a look tomorrow!